compiler/src/dmd/expression.d

```diff
@@ -2494,13 +2494,13 @@ extern (C++) final class SuperExp : ThisExp
         super(loc, EXP.super_);
     }
 
-    override final bool isLvalue()
+    override bool isLvalue()
     {
         // Class `super` should be an rvalue
         return false;
     }
 
-    override final Expression toLvalue(Scope* sc, Expression e)
+    override Expression toLvalue(Scope* sc, Expression e)
```

Comment:
This looks to be unrelated.

Reply:
It is, but it is a complaint from from Dstyle that was overlooked before.

Reply:
https://github.com/dlang/dmd/pull/15395